docs/user-guide/prereqs.md

```diff
@@ -7,11 +7,41 @@ assignees:
 
 To deploy and manage applications on Kubernetes, you’ll use the Kubernetes command-line tool, [kubectl](/docs/user-guide/kubectl/). It lets you inspect your cluster resources, create, delete, and update components, and much more. You will use it to look at your new cluster and bring up example apps.
 
-## Installing kubectl
+## Downloading kubectl
+
+If you built from source, kubectl should be either under `_output/local/bin/<os>/<arch>` or `_output/dockerized/bin/<os>/<arch>`.
 
 If you downloaded a pre-compiled [release](https://github.com/kubernetes/kubernetes/releases), kubectl should be under `platforms/<os>/<arch>` from the tar bundle.
 
-If you built from source, kubectl should be either under `_output/local/bin/<os>/<arch>` or `_output/dockerized/bin/<os>/<arch>`.
+You may also download a pre-built kubectl client binary for say `${K8S_VERSION}` (in this example: `{{page.version}}.0`) for specific platforms:
+
+```shell
```

Comment:
This part is the same as http://kubernetes.io/docs/getting-started-guides/minikube/#install-kubectl

@devin-donnelly should we duplicate it or put this in a centralized place?